Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+
+[packages]
+aiohttp = "*"
+"discord.py" = {git = "https://github.com/rapptz/discord.py", ref = "rewrite"}
+scrapy = "*"
+numpy = "*"
+
+[dev-packages]
+pylint = "*"
```

Comment:
You should avoid star dependencies. Try and get a version, even if it's only a major version. The only exception to that is maybe pylint.